app/models/vm_or_template/scanning.rb

```diff
@@ -8,6 +8,12 @@
 module VmOrTemplate::Scanning
   extend ActiveSupport::Concern
 
+  # Smartstate Analysis is unsupported by default.
+  # Subclasses need to override this method if they support SSA.
+  included do
+    supports_not :smartstate_analysis, :reason => _("Smartstate Analysis is not available for VM or Template.")
```

Comment:
@roliveri  This was discussed with @durandom and he preferred to have these details removed because the text is only displayed on the disabled buttons.